### ISSUE TITLE
Build failed with error NETSDK1004 for F# ConsoleApp 

### ISSUE THREAD

**Issue by Jepson109** [Microsoft/visualfsharp#6412]
#### Repro steps

1. Create a new F# Console application via VS

2. Restore & Build

#### Expected behavior

Restore and Build succeeded 
![image](https://user-images.githubusercontent.com/38854445/55389013-fd86b600-5566-11e9-88af-a22f68cb70d7.png)

#### Actual behavior
1.Build failed
` error NETSDK1004: Assets file 'C:\Users\e2euser\Source\Repos\ConsoleApp13\obj\project.assets.json' not found. Run a NuGet package restore to generate this file.`
![image](https://user-images.githubusercontent.com/38854445/55384779-2eaeb880-555e-11e9-99ae-bd1c48afd19b.png)
2.Dependencies are missing
![image](https://user-images.githubusercontent.com/38854445/55385173-1db27700-555f-11e9-8afe-18c24a32f333.png)
3.It should show the project name  
![image](https://user-images.githubusercontent.com/38854445/55385686-45eea580-5560-11e9-8839-66f6b259fa92.png)
Note: Create a new F# Console application via CLI, restore, build and run succeeded.
#### Related information 

```
C:\Users\e2euser>dotnet --info
.NET Core SDK (reflecting any global.json):
 Version:   3.0.100-preview4-011024
 Commit:    ac32f7536d

Runtime Environment:
 OS Name:     Windows
 OS Version:  6.1.7601
 OS Platform: Windows
 RID:         win7-x64
 Base Path:   C:\Program Files\dotnet\sdk\3.0.100-preview4-011024\

Host (useful for support):
  Version: 3.0.0-preview4-27601-05
  Commit:  97567a0e5f

.NET Core SDKs installed:
  2.1.602 [C:\Program Files\dotnet\sdk]
  2.2.202 [C:\Program Files\dotnet\sdk]
  3.0.100-preview4-011024 [C:\Program Files\dotnet\sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.All 2.1.9 [C:\Program Files\dotnet\shared\Microsoft.AspNe
tCore.All]
  Microsoft.AspNetCore.All 2.2.3 [C:\Program Files\dotnet\shared\Microsoft.AspNe
tCore.All]
  Microsoft.AspNetCore.App 2.1.9 [C:\Program Files\dotnet\shared\Microsoft.AspNe
tCore.App]
  Microsoft.AspNetCore.App 2.2.3 [C:\Program Files\dotnet\shared\Microsoft.AspNe
tCore.App]
  Microsoft.AspNetCore.App 3.0.0-preview4-19180-01 [C:\Program Files\dotnet\shar
ed\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.1.9 [C:\Program Files\dotnet\shared\Microsoft.NETCore.
App]
  Microsoft.NETCore.App 2.2.3 [C:\Program Files\dotnet\shared\Microsoft.NETCore.
App]
  Microsoft.NETCore.App 3.0.0-preview4-27601-05 [C:\Program Files\dotnet\shared\
Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 3.0.0-preview4-27529-17 [C:\Program Files\dotnet\
shared\Microsoft.WindowsDesktop.App]

To install additional .NET Core runtimes or SDKs:
  https://aka.ms/dotnet-download
```

**Comment by cartermp** [comment]
@Jepson109 .NET Core 3.0 in VS 2019 is a bit complicated. The new project dialog will show templates from the SDK if you have it installed, but that SDK will not actually be built against in VS unless you explicitly check the setting to use Preview SDKs.

![image](https://user-images.githubusercontent.com/6309070/55413601-7c0b3580-551e-11e9-969e-f1c89ed2b11c.png)

If you have that checked and this reproduces then we can investigate.

The error message indicates an issue in the SDK and not F# tooling, but it should all "just work".

**Comment by richaverma1** [comment]
@cartermp, this was tested against D16.1/28801.116 (16.1.0 preview 1.0) and hence the checkbox should be checked automatically. @Jepson109 , please make sure to also provide the VS information in the bug.

**Comment by cartermp** [comment]
Thanks! We'll take a look

**Comment by Jepson109** [comment]
@richaverma1, the checkbox has been  checked automatically and the VS information is shown below. 
![image](https://user-images.githubusercontent.com/38854445/55446136-fb693980-55f0-11e9-9768-1deb71647f5e.png)

**Comment by cartermp** [comment]
I've been able to reproduce on 16.1 builds. F# and .NET SDK-style projects aren't getting restored.

**Comment by cartermp** [comment]
Tracking here for visibility. We believe there is a bug in the project system causing .NET SDK-style F# projects to load with the old F# project system, which naturally doesn't work.

**Comment by abelbraaksma** [comment]
@cartermp, I just hit this as well (for Core 2.2). In fact, it happened after loading an VS2017-created hello-world .NET Core Console application into VS2019. 

Assuming I messed up (it showed empty as in the screenshot above), I deleted it and recreated it using all the defaults for an F# Console .NET Core application. Trying to run failed with above message.

This is indeed VS 16.1 Preview 1 Enterprise. However, the OP shows .NET Core 3.0, but mine shows .NET Core 2.2:

```
.NET Core SDK (reflecting any global.json):
 Version:   2.2.300-preview-010046
 Commit:    d6f4b3dc39

Runtime Environment:
 OS Name:     Windows
 OS Version:  6.1.7601
 OS Platform: Windows
 RID:         win7-x64
 Base Path:   C:\Program Files\dotnet\sdk\2.2.300-preview-010046\

Host (useful for support):
  Version: 2.2.3
  Commit:  6b8ad509b6
```

Is there a workaround for this? I'm experimenting with some commands, but can't find it yet.
